app/services/container_dashboard_service.rb

```diff
@@ -259,7 +259,11 @@ def daily_image_metrics
   end
 
   def daily_provider_metrics
-    @daily_metrics ||= Metric::Helper.find_for_interval_name('daily', @controller.current_user.get_timezone)
+    current_user = @controller.current_user
+    tp = TimeProfile.profile_for_user_tz(current_user.id, current_user.get_timezone)
+    tp = TimeProfile.default_time_profile if tp.nil?
```

Comment:
the Ruby idiom is:
`tp ||= TimeProfile.default_time_profile`

Alternatively, it could be:
`tp = TimeProfile.profile_for_user_tz(current_user.id, current_user.get_timezone) || TimeProfile.default_time_profile`